lalrpop/src/parser/lrgrammar.lalrpop

```diff
@@ -92,18 +107,20 @@ Alternatives: Vec<Alternative> = {
 };
 
 Alternative: Alternative = {
-    <lo:@L> <s:Symbol+> <c:("if" <Cond>)?> <a:Action?> <hi:@R> => {
+    <lo:@L> <n:Annotation*> <s:Symbol+> <c:("if" <Cond>)?> <a:Action?> <hi:@R> => {
```

Comment:
Pre-existing, but now that inlining works in the snapshot we should be able to remove the second arm here and change `Symbol+` into `Symbol*`.